lang/en/docs/cli/install.md

```diff
@@ -84,9 +87,58 @@ Shallowly installs a packages's sibling workspace dependencies underneath its `n
 
 Must be run inside an individual workspace in a workspaces project. Can not be run in a non-workspaces project or at the root of a workspaces project.
 
+
 ##### `yarn install --frozen-lockfile` <a class="toc" id="toc-yarn-install-frozen-lockfile" href="#toc-yarn-install-frozen-lockfile"></a>
 
-Don't generate a `yarn.lock` lockfile and fail if an update is needed.
+Ensure that `yarn.lock` is in sync with `package.json`, returning an error if it
+is not, and then act as if `--pure-lockfile` ([see
+above](#toc-yarn-install-pure-lockfile)) were passed.
+
+"In sync" does not mean that yarn will throw an error if a dependency is out of
+date.  It will only throw an error if `yarn.lock` is incompatible with
+`package.json`.  For example, if a new package were added to `package.json` but
+`yarn` had not been run; or if the version in `yarn.lock` were to not match the
+version range specified in `package.json`.
+
+###### Concrete example
+
+    What if 2.4.5 is released by the vue maintainers and no changes are made the
```

Comment:
This needs the context from the [previous comment](https://github.com/yarnpkg/website/issues/700#issuecomment-341135505) - ie
> packages.json dependency:
> { "dependencies": { "vue": "^2.2.6", } }
>
> Resulting lockfile dependency:
> vue@^2.2.6: version "2.4.4" resolved "https://registry.yarnpkg.com/vue/-/vue-2.4.4.tgz#ea9550b96a71465fd2b8b17b61673b3561861789"